onvm/go.sh

```diff
@@ -36,6 +36,8 @@ nf_cores=$3
 SCRIPT=$(readlink -f "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
 verbosity=1
+#default base virtual address set
+virt_addr="--base-virtaddr=0x7f000000000"
 
```

Comment:
there is an echo output in `function usage {` saying that it adds a virt_addr parameter. we should change the wording to make it clear that there is a base_virt addr default.